scripts/python/proposers_rewards.py

```diff
@@ -0,0 +1,416 @@
+from typing import Dict, Optional, List, Tuple, Generator, TextIO, Union
+
+import sys
+import asyncio
+import json
+import itertools
+import enum
+import os
+from collections import namedtuple
+
+import pydantic
+import httpx
+import typer
+
+# VIT servicing station models
+
+ADA = '₳'
+FUNDED = "FUNDED"
+NOT_FUNDED = "NOT_FUNDED"
+YES = "YES"
+NO = "NO"
+NOT_FUNDED_OVER_BUDGET = "Not Funded - Over Budget"
+NOT_FUNDED_APPROVAL_THRESHOLD = "Not Funded - Approval Threshold"
+LOVELACE_FACTOR = 1000000
+
+
+class Proposal(pydantic.BaseModel):
+    internal_id: int
+    proposal_id: str
+    proposal_title: str
+    proposal_funds: int
+    proposal_url: str
+    chain_proposal_id: str
+    chain_proposal_index: int
+    chain_vote_options: Dict[str, int]
+    fund_id: int
+    challenge_id: int
+    challenge_type: str
+
+
+# Jormungandr models
+
+class Options(pydantic.BaseModel):
+    start: int
+    end: int
+
+
+class Results(pydantic.BaseModel):
+    results: List[int]
+
+
+# we mimic the tally
+class TallyResult(pydantic.BaseModel):
+    result: Results
+
+
+class DecryptedTally(pydantic.BaseModel):
+    decrypted: TallyResult
+
+
+class PrivateTallyState(pydantic.BaseModel):
+    state: DecryptedTally
+
+
+class PrivateTally(pydantic.BaseModel):
+    Private: PrivateTallyState
```

Comment:
just curious, why is this uppercase? (`Private` instead of `private`)

Reply:
in the talley result it is defined as uppercase:

```json
"tally": {
          "Public": {
```